nixos/modules/services/hardware/logiops.nix

```
@@ -0,0 +1,77 @@
+{ config, lib, pkgs, ... }:
+
+let
+  inherit (lib) mkEnableOption mkIf mkOption types;
+  cfg = config.services.logiops;
+  format = pkgs.formats.libconfig { };
+in {
+  options.services.logiops = {
+    enable = mkEnableOption
+      "logiops, an unofficial userspace driver for HID++ Logitech devices";
+
+    settings = mkOption {
+      type = types.submodule { freeformType = format.type; };
+      description = ''
+        Configuration for logiops.
+
+        See <https://github.com/PixlOne/logiops/wiki/Configuration> for more details.
+        Also see <https://github.com/PixlOne/logiops/blob/main/logid.example.cfg> for an example config.
+      '';
+      default = { };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    environment.etc."logid.cfg".source =
+      format.generate "logid.cfg" cfg.settings;
+    systemd.services.logiops = {
+      description = "Logitech Configuration Daemon";
+      documentation = [ "https://github.com/PixlOne/logiops" ];
+
+      wantedBy = [ "graphical.target" ];
```

```suggestion
      wantedBy = [ "multi-user.target" ];
```
The mouse can be configured before the system is done booting and the display-manager has started. It's a nice sign of life.